usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -50,12 +53,20 @@ def handle(self, *args, **options):
             return
 
         failed_submissions = []
-        for submission_id in submission_ids:
+        submission_ids = deque(submission_ids)
+        while submission_ids:
+            submission_id = submission_ids.popleft()
             try:
                 call_command("load_submission", submission_id)
             except SystemExit:
-                logger.info(f"Submission failed to load: {submission_id}")
+                logger.info(f"Submission {submission_id} failed to load")
                 failed_submissions.append(submission_id)
+                # This is a system exit so we really shouldn't be swallowing it.  Let's log a little additional
+                # information and re-raise the exception.
+                logger.info("Ending execution early due to SystemExit")
+                logger.info(f"{len(failed_submissions):,} submission failures occurred: {failed_submissions}")
+                logger.info(f"{len(submission_ids):,} submissions remain unprocessed: {list(submission_ids)}")
+                raise
```

Comment:
There has been a lingering effort to allow other submissions to be loaded before terminating the script due to a problematic submission. Logging the issues are great, but why not let the script continue onto other submissions and then terminate with a non-success status?

Reply:
I didn't see `SystemExit` or `exit` being used anywhere in `load_submissions` so I figured if we got one something must have gone really really wrong, but it's been working the other way for a long time...

I've reversed the change... with a twist.  👍